docs/UserGuide.md

```diff
@@ -158,147 +128,241 @@ Examples:
+
+
+### Editing a student : `editPerson` 
 
 Edits an existing student in the application.
 
-Format: `edit INDEX [-name NAME] [-phone PHONE_NUMBER] [-email EMAIL] [-address ADDRESS] [-tag TAG]`
+Format: `editPerson INDEX [-name NAME] [-phone PHONE_NUMBER] [-email EMAIL] [-address ADDRESS] [-subject SUBJECT] [-tag TAG] [-remark REMARK]` <br>
 
 * Edits the student at the specified `INDEX`. The index refers to the index number shown in the displayed student list. The index **must be a positive integer** 1, 2, 3, …​
+* If the user is currently in list `STUDENTS`, the command can be shortened to `edit INDEX`
+* If the user is currently in list `STUDENTS` with a student shown, the index can be omitted and the command can be shortened to `edit`. The shown student will be edited.
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the student will be removed i.e. adding of tags is not cumulative.
+* For flags that can take multiple values (eg. -subject, -tag), separate the values with commas
+* When editing subjects, tags and remarks, the existing subjects/tags/remarks of the student will be removed i.e. adding of subjects/tags/remarks is not cumulative.
+* Subjects can only be MATHEMATICS, PHYSICS, BIOLOGY, CHEMISTRY or ENGLISH.
 
 Examples:
-*  `edit 1 -phone 91234567 -email johndoe@example.com` Edits the phone number and email address of the 1st student to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 -name Betsy Crower -tag Secondary` Edits the name of the 2nd student to be `Betsy Crower` and sets the tag to `Secondary`.
+*  `editPerson 1 -phone 91234567 -email johndoe@example.com` <br> Edits the phone number and email address of the 1st student to be `91234567` and `johndoe@example.com` respectively.
+*  `editPerson 2 -name Betsy Crower -tag Secondary`<br> Edits the name of the 2nd student to be `Betsy Crower` and sets the tag to `Secondary`.
+*  `editPerson 3 -subject chemistry, biology` <br> Sets the subjects of the 3rd student to be `CHEMISTRY` and `BIOLOGY`.
+* In list `STUDENTS`:
+  * `edit 1 -name John -phone 91234567` <br> Edits the name and phone number of the 1st student to be `John` and `91234567` respectively.
+* In list `STUDENTS` and 2nd student is shown:
+  * `edit -subject chemistry, english` <br> Edits the subject of the 2nd lesson to be `CHEMISTRY` and `ENGLISH`
 
-### Locating students by name: `find`
 
-The find command has different behaviours depending on the current list:
+### Adding a Lesson : `addLesson`
 
-1. In `STUDENTS` list:
-    - Finds students whose names are made up of the given search keyword.
-2. In `SCHEDULE` list:
-    - Finds lessons whose names are made up of the given search keyword.
+Adds a lesson to the schedule list in application.
 
-Format: `find KEYWORD`
+Format: `addLesson -name NAME [-start HH:MM] [-end HH:MM] [-day YYYY/MM/DD] [-subject SUBJECT]` <br>
```

Comment:
probably [-day YYYY/MM/DD or YY/MM/DD or MM/DD or DD] is more concise

Reply:
Made the necessary changes in recent commits. Thank you!